libs/backend/src/system_call/export_logs_to_usb.ts

```diff
@@ -49,6 +49,7 @@ async function exportLogsToUsbHelper({
   try {
     await execFile('mkdir', ['-p', machineNamePath]);
     await execFile('cp', ['-r', LOG_DIR, destinationDirectory]);
+    await execFile('sync', [status.mountPoint]);
```

Comment:
I know elsewhere we use `sync -f ...`, e.g.

https://github.com/votingworks/vxsuite/blob/d48791757a5ad2e15c5980b2cd7320ec5d6c557e/libs/backend/src/exporter.ts#L159-L161

```
$ sync --help
Usage: sync [OPTION] [FILE]...
Synchronize cached writes to persistent storage

If one or more files are specified, sync only them,
or their containing file systems.

  -d, --data             sync only file data, no unneeded metadata
  -f, --file-system      sync the file systems that contain the files
      --help        display this help and exit
      --version     output version information and exit
```

Just double checking that we don't need that here. I don't _think_ we do since `status.mountPoint` is the root of the USB drive file system.

Reply:
I'm confident we don't need `-f` for the reason you pointed it, but I added it in for consistency with the `exporter` code.